command/results_gotest.go

```diff
@@ -52,15 +52,15 @@ func (c *goTestResults) Execute(ctx context.Context,
 		return err
 	}
 
-	// make sure the file patterns are relative to the task's working directory
-	for idx, file := range c.Files {
-		c.Files[idx] = filepath.Join(conf.WorkDir, file)
+	// All file patterns should be relative to the task's working directory.
+	for i, file := range c.Files {
+		c.Files[i] = filepath.Join(conf.WorkDir, file)
 	}
 
 	// will be all files containing test results
-	outputFiles, err := c.allOutputFiles()
+	outputFiles, err := globFiles(c.Files...)
 	if err != nil {
-		return errors.Wrap(err, "error obtaining names of output files")
+		return errors.Wrap(err, "obtaining names of output files")
```

Comment:
Yeah I don't think error should be removed here. Error messages should be sentences.

Reply:
Same thing as above.